src/Makefile-rocksdb.am

```diff
@@ -1,846 +1 @@
-  rocksdb/utilities/write_batch_with_index/write_batch_with_index_internal.cc \
-  rocksdb/utilities/write_batch_with_index/write_batch_with_index_internal.h \
-  rocksdb/utilities/write_batch_with_index/write_batch_with_index_test.cc \
-  rocksdb/util/kv_map.h \
-  rocksdb/util/log_buffer.cc \
-  rocksdb/util/log_buffer.h \
-  rocksdb/util/logging.cc \
-  rocksdb/util/logging.h \
-  rocksdb/util/log_write_bench.cc \
-  rocksdb/util/lru_cache_handle.h \
-  rocksdb/util/memenv.cc \
-  rocksdb/util/mock_env.cc \
-  rocksdb/util/mock_env.h \
-  rocksdb/util/mock_env_test.cc \
-  rocksdb/util/murmurhash.cc \
-  rocksdb/util/murmurhash.h \
-  rocksdb/util/mutable_cf_options.cc \
-  rocksdb/util/mutable_cf_options.h \
-  rocksdb/util/mutexlock.h \
-  rocksdb/util/options_builder.cc \
-  rocksdb/util/options.cc \
-  rocksdb/util/options_helper.cc \
-  rocksdb/util/options_helper.h \
-  rocksdb/util/options_parser.cc \
-  rocksdb/util/options_parser.h \
-  rocksdb/util/options_sanity_check.cc \
-  rocksdb/util/options_sanity_check.h \
-  rocksdb/util/options_settable_test.cc \
-  rocksdb/util/options_test.cc \
-  rocksdb/util/perf_context.cc \
-  rocksdb/util/perf_context_imp.h \
-  rocksdb/util/perf_level.cc \
-  rocksdb/util/perf_level_imp.h \
-  rocksdb/util/perf_step_timer.h \
-  rocksdb/util/posix_logger.h \
-  rocksdb/util/random.cc \
-  rocksdb/util/random.h \
-  rocksdb/util/rate_limiter.cc \
-  rocksdb/util/rate_limiter.h \
-  rocksdb/util/rate_limiter_test.cc \
-  rocksdb/util/slice.cc \
-  rocksdb/util/slice_transform_test.cc \
-  rocksdb/util/sst_file_manager_impl.cc \
-  rocksdb/util/sst_file_manager_impl.h \
-  rocksdb/util/statistics.cc \
-  rocksdb/util/statistics.h \
-  rocksdb/util/status.cc \
-  rocksdb/util/status_message.cc \
-  rocksdb/util/stderr_logger.h \
-  rocksdb/util/stop_watch.h \
-  rocksdb/util/string_util.cc \
-  rocksdb/util/string_util.h \
-  rocksdb/util/sync_point.cc \
-  rocksdb/util/sync_point.h \
-  rocksdb/util/testharness.cc \
-  rocksdb/util/testharness.h \
-  rocksdb/util/testutil.cc \
-  rocksdb/util/testutil.h \
-  rocksdb/util/thread_list_test.cc \
-  rocksdb/util/thread_local.cc \
-  rocksdb/util/thread_local.h \
-  rocksdb/util/thread_local_test.cc \
-  rocksdb/util/thread_operation.h \
-  rocksdb/util/threadpool.cc \
-  rocksdb/util/threadpool.h \
-  rocksdb/util/thread_status_impl.cc \
-  rocksdb/util/thread_status_updater.cc \
-  rocksdb/util/thread_status_updater_debug.cc \
-  rocksdb/util/thread_status_updater.h \
-  rocksdb/util/thread_status_util.cc \
-  rocksdb/util/thread_status_util_debug.cc \
-  rocksdb/util/thread_status_util.h \
-  rocksdb/util/transaction_test_util.cc \
-  rocksdb/util/transaction_test_util.h \
-  rocksdb/util/xfunc.cc \
-  rocksdb/util/xfunc.h \
-  rocksdb/util/xxhash.cc \
-  rocksdb/util/xxhash.h \
-  rocksdb/Vagrantfile \
-  rocksdb/WINDOWS_PORT.md \
-  rocksdb/AUTHORS
+EXTRA_DIST += $(find rocksdb/ -type f)
```

Comment:
probably you can reference the implementation in https://github.com/ceph/ceph/blob/master/src/script/make_rocksdb_makefile.sh ? or revise it so we can leverage it here?